README.md

```diff
@@ -1,5 +1,3 @@
-[![Build Status](https://travis-ci.org/google/google-api-dotnet-client.svg?branch=master)](https://travis-ci.org/google/google-api-dotnet-client)
```

Comment:
I'm sorry, I'm still not an expert on GitHub. My recommendation for you is to just revert those files, it should be easy. 
Because your CL is pretty small, you can also just create a new PR.
Up to you :+1: